code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -1419,3 +1419,24 @@
 	icon_state = "stamp-midhorror"
 	stamptext = "This paper has been certified by The Council of Mid Horror"
 
+//thedavestdave Lucky
+///I know this is pretty bodgey but if it stupid and it works it isn't stupid
+/obj/item/clothing/suit/storage/hooded/explorer/Lucky
```

Comment:
Minor standards thing, but you should avoid capitalization in item paths and icon/item state names.

Reply:
low case l installed